src/utils/error-to-string.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function errorToString(err) {
+  const isString = typeof error === 'string';
```

Comment:
can we use lodash's _.isString? This is for consistency with other bits in the codebase

Reply:
changed